epub33/core/index.html

```diff
@@ -5963,6 +5963,15 @@ <h4>URLs in the OCF Abstract Container</h4>
 						of <a data-cite="url#concept-url-parser">parsing</a> the file's <a>File Path</a> with the
 							<a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a>.</p>
 
+					<div class="note">
+						The constraints on a <a>container root URL</a> are automatically honored for "usual" URLs like,
+						for example,  <code>https://localhost:12345/</code> or <code>https://www.example.org:12345/</code>; however,
+						a Reading System may choose a different strategy to identify the files of an EPUB container like, for example,
+						<code>file://path/to.epub#path=/EPUB/content.opf</code> or <code>jar:file:/path/to.epub!/EPUB/content.opf</code>.
+						In such a case, the Reading System has to implement its own version of URL parsing to conform to the conceptual
+						model of a <a>container root URL</a> as defined in this specification.
```

Comment:
Well... In theory I could imagine using these URL-s and then the RS would have to re-write the URL parsing library to abide to the behavioural specification of URL-s. Becuse the container root URL-s are, in some way, hidden to the everyday user (e.g., for RS-s that do not use scripting), this could be done and thereby making the RS conformant. I would agree that it would be mad to do it this way, but I shied away to declare these RS-s non-conformant.

That being said, it makes it probably easier to read, in a note, that these are non-conformant, even though a specific RS may find a way around it...

Reply:
Looking at your generic reformulation, though, this issue is becoming moot anyway...